compiler/rustc_builtin_macros/src/lib.rs

```diff
@@ -48,7 +48,7 @@ pub fn register_builtin_macros(resolver: &mut dyn ResolverExpand, edition: Editi
     let mut register = |name, kind| {
         resolver.register_builtin_macro(
             Ident::with_dummy_span(name),
-            SyntaxExtension { is_builtin: true, ..SyntaxExtension::default(kind, edition) },
+            SyntaxExtension { builtin_name: Some(name), ..SyntaxExtension::default(kind, edition) },
```

Comment:
```suggestion
            SyntaxExtension::default(kind, edition)
```
Setting `is_builtin` here was necessary in the past, but it shouldn't be necessary now.